.gitignore

```diff
@@ -1 +1,17 @@
-.DS_Store
```

Comment:
Why did you delete this from your .gitignore? It's definitely a good thing to have in here. 👍